ts/components/NewRemindEmailValidationOverlay.tsx

```diff
@@ -0,0 +1,192 @@
+/**
+ * A component to remind the user to validate his email
+ */
+import { Millisecond } from "@pagopa/ts-commons/lib/units";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import { Content } from "native-base";
+import * as React from "react";
+import { useCallback, useEffect, useRef, useState } from "react";
+import { View, SafeAreaView } from "react-native";
+import {
+  ButtonLink,
+  IOPictogramSizeScale,
+  Label,
+  Pictogram,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { useNavigation } from "@react-navigation/native";
+import I18n from "../i18n";
+
+import {
+  profileLoadRequest,
+  startEmailValidation
+} from "../store/actions/profile";
+import {
+  isProfileEmailValidatedSelector,
+  profileEmailSelector
+} from "../store/reducers/profile";
+import { useIODispatch, useIOSelector } from "../store/hooks";
+import ROUTES from "../navigation/routes";
+import { IOStyles } from "./core/variables/IOStyles";
+import FooterWithButtons from "./ui/FooterWithButtons";
+import { IOToast } from "./Toast";
+import BaseScreenComponent from "./screens/BaseScreenComponent";
+
+const emailSentTimeout = 10000 as Millisecond; // 10 seconds
+const profilePolling = 5000 as Millisecond; // 5 seconds
+
+const EMPTY_EMAIL = "";
+const VALIDATION_ILLUSTRATION_WIDTH: IOPictogramSizeScale = 80;
+
+const NewRemindEmailValidationOverlay = () => {
+  const dispatch = useIODispatch();
+  const optionEmail = useIOSelector(profileEmailSelector);
+  const isEmailValidated = useIOSelector(isProfileEmailValidatedSelector);
+  const [isVerifyButtonDisabled, setIsVerifyButtonDisabled] = useState(false);
```

Comment:
Would `sendVerificationEmailButton` or `validateEmailButton` be a better name in this case?

Reply:
done